standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -367,6 +368,7 @@ public static void startMetaStore(int port, HadoopThriftAuthBridge bridge,
     boolean tcpKeepAlive = MetastoreConf.getBoolVar(conf, ConfVars.TCP_KEEP_ALIVE);
     boolean useCompactProtocol = MetastoreConf.getBoolVar(conf, ConfVars.USE_THRIFT_COMPACT_PROTOCOL);
     boolean useSSL = MetastoreConf.getBoolVar(conf, ConfVars.USE_SSL);
+    ProxyUsers.refreshSuperUserGroupsConfiguration(conf);
```

Comment:
is done somewhere else implicitly before hadoop 3.3?

Reply:
So ya, this was done as a separate thing buried in the Hive code.  This makes it much more explicit and less hidden.

Before Hadoop 3.3, it could easily be detected if a call to `refreshSuperUserGroupsConfiguration` had already been performed because there was a corresponding getter that would return a `null` value if it had not.  Well, in 3.3 that went away and instead of returning null, you get some sort of default value.  So now one can't lazily refresh these configurations, if they haven't already been, it's better to just refresh them explicitly here and be done with it.